app/src/main/java/com/einmalfel/podlisten/Provider.java

```diff
@@ -137,6 +137,10 @@ public long getMaxAge() {
   private static HelperV1 helper;
```

Comment:
this "helper" field is accessed from different threads, so it's a bug that it is not volatile. Accessing it via synchronized setters&getters will fix it too. But what is the reason for adding non-synchronized setter?